src/coreclr/nativeaot/Runtime/Full/CMakeLists.txt

```diff
@@ -60,6 +62,7 @@ if (CLR_CMAKE_TARGET_WIN32)
   add_library(Runtime.ServerGC.GuardCF STATIC ${COMMON_RUNTIME_SOURCES} ${FULL_RUNTIME_SOURCES} ${RUNTIME_SOURCES_ARCH_ASM} ${SERVER_GC_SOURCES} ${RUNTIME_ARCH_ASM_OBJECTS})
   target_compile_definitions(Runtime.ServerGC.GuardCF PRIVATE -DFEATURE_SVR_GC)
   set_target_properties(Runtime.ServerGC.GuardCF PROPERTIES CLR_CONTROL_FLOW_GUARD ON)
+  target_link_libraries(Runtime.ServerGC.GuardCF PRIVATE minipal_objects)
```

Comment:
Does CMake flow `CLR_CONTROL_FLOW_GUARD ON` to the minipal sources automagically?

Reply:
Good catch, I'll set that up.